JackieBinya/Free-Mentors-
# (169252515) Modifies constraints on users table

#### What does this PR do?
- modifies  server/src/v1/db/createTables.js

#### Description of Task to be completed
- it removes the **NOT NULL** constraint from the columns _address_, _bio_,  _expertise_, and _occupation_ from the **users** table.


#### How should this be manually tested?
- On the terminal run, the command ```npm  run dropTables```, to drop any tables that might exist in your database.
- Then run the command ```npm run createTables``` to create both the **users** and **sessions** table.
- Open *pgAdmin*, refresh your database, then view your ** users** table columns and their constraints.
- As expected the columns _address_, _bio_,  _expertise_, and _occupation_ from the **users** table should not have the constraint **NOT NULL**

#### Any background context you want to provide?
- I am refactoring my API such that upon sign up its not mandatory for a user to provide _address_, _bio_, _expertise_ and _occupation_.

#### What are the relevant pivotal tracker stories?
[#169252515](https://www.pivotaltracker.com/n/projects/2382168)

#### Screenshots (if appropriate)
